Source/WebCore/WebCoreMacros.cmake

```
@@ -189,6 +189,19 @@ macro(GENERATE_EVENT_FACTORY _infile _namespace)
 endmacro()
 
 
+macro(GENERATE_EVENT_NAMES _infile _namespace)
+    set(NAMES_GENERATOR ${WEBCORE_DIR}/dom/make-event-names.py)
+    set(_outputfiles ${WebCore_DERIVED_SOURCES_DIR}/${_namespace}EventNames.h ${WebCore_DERIVED_SOURCES_DIR}/${_namespace}EventNames.cpp)
```

I think the bug on CMake-based platforms is that this line needs to change because the generated code will create filenames of `EventEventNames.h` and `EventEventNames.cpp` the way it's currently written.

Maybe the `_namespace` argument isn't even needed for this code generation?

```suggestion
    set(_outputfiles ${WebCore_DERIVED_SOURCES_DIR}/${_namespace}Names.h ${WebCore_DERIVED_SOURCES_DIR}/${_namespace}Names.cpp)
```

---

Nice catch. Thanks.